src/PasswordConstraintPluginManager.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Drupal\password_enhancements;
+
+use Drupal\Core\Cache\CacheBackendInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\Core\Plugin\DefaultPluginManager;
+use Traversable;
+
+/**
+ * Password constraint plugin manager.
+ */
+class PasswordConstraintPluginManager extends DefaultPluginManager {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(Traversable $namespaces, CacheBackendInterface $cache_backend, ModuleHandlerInterface $module_handler) {
+    parent::__construct(
+      'Plugin/PasswordConstraint',
+      $namespaces,
+      $module_handler,
+      'Drupal\password_enhancements\Plugin\PasswordConstraintPluginInterface',
```

Comment:
This can be replaced with a class constant.

Reply:
Okay, using ::class sounds better.